link.T

```diff
@@ -63,20 +63,18 @@ NORELOC_BSS_END_ = .;
     *(.hash)
   }
 
-  .eh_frame_hdr : {
-    __eh_frame_hdr_start = .;
-    __eh_frame_hdr_start2 = ALIGN(0x10);
-    *(.eh_frame_hdr)
-    __eh_frame_hdr_end = .;
-    __eh_frame_hdr_end2 = ALIGN(0x10);
-  }
-
-  .eh_frame : {
+  .eh_frame : ALIGN(0x1000) {
     __eh_frame_start = .;
     *(.eh_frame)
     __eh_frame_end = .;
   }
 
+  .eh_frame_hdr : ALIGN(0x1000) {
+    __eh_frame_hdr_start = .;
+    *(.eh_frame_hdr)
+    __eh_frame_hdr_end = .;
+  }
+
```

Comment:
These sections still don't make it into the final NRO. They get discarded by elf2nxo.py

Reply:
Interesting, so the runtime errors we were seeing previously were simply an alignment issue, probably throwing _everything_ out of wack.

I assume this is some issue with elf2nxo, or nxodance?